homeassistant/components/dnsip/sensor.py

```diff
@@ -88,8 +89,8 @@ def __init__(
         self._attr_name = "IPv6" if ipv6 else None
         self._attr_unique_id = f"{hostname}_{ipv6}"
         self.hostname = hostname
-        self.resolver = aiodns.DNSResolver(tcp_port=port, udp_port=port)
-        self.resolver.nameservers = [resolver]
+        self.port = port
+        self._resolver = resolver
```

Comment:
It might be confusing to have two resolvers of different types. Maybe it makes sense to rename this and the param to something like `nameserver` instead?